annotation-utils/src/main/java/edu/isi/vista/annotationutils/AceCorpusTextSource.java

```diff
@@ -0,0 +1,52 @@
+package edu.isi.vista.annotationutils;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import edu.isi.nlp.symbols.Symbol;
+import com.google.common.base.Optional;
+import edu.isi.nlp.io.OriginalTextSource;
+import java.io.File;
+import java.nio.file.Path;
+import java.io.IOException;
+
+
+public final class AceCorpusTextSource implements OriginalTextSource{
+    private  File aceDataDirectory;
+    private File[] sourceFileDirectories;

```

Comment:
style: Unless you are writing low-level code, you almost never want to use arrays in Java. Prefer `List<File>` (or `List<Path>`).